src/Frontend/Core/Engine/Header.php

```diff
@@ -132,7 +132,11 @@ public function addCSS($file, $minify = true, $addTimestamp = null)
         $file = (string) $file;
         $minify = (bool) $minify;
         $addTimestamp = (bool) $addTimestamp;
-        $file = Theme::getPath($file);
+
+        // get file path
+        if (mb_substr($file, 0, 4) != 'http') {
```

Comment:
I find it better matching with what you try to test
because if you read the code it is check if the string starts with http
so I think strpos is better suited for that